### ISSUE TITLE
Airdrop farmer addresses with identical txns

### ISSUE THREAD

**Issue by p3rcy-eth** [safe-global/safe-user-allocation-reports#142]

**Comment by tschubotz** [comment]
This report is missing the "reasoning" section as required by the template.

> Then, they created totally 4 safes and made 0 value transactions

All 4 Safes make txs with value >0

You reasoning only considers how the 2 EOAs are connected, not how the Safes are created for Airdrop farming.

**Comment by p3rcy-eth** [comment]
> This report is missing the "reasoning" section as required by the template.
> 
> > Then, they created totally 4 safes and made 0 value transactions
> 
> All 4 Safes make txs with value >0
> 
> You reasoning only considers how the 2 EOAs are connected, not how the Safes are created for Airdrop farming.

Goodday ser, I found that I was a bit late, and issue #384 used the same approach as mine. I'm here to facilitate his/her report.
Persuant to my initial claim that there are two addresses that made identical txns and have created 4 safes together, I thought that using the withdraws from FTX Exchange 2 within a certain time period may be a good idea, because that may give me more airdrop farming addressess. So I used the following SQL in dune:
```
select * from ethereum."transactions" where block_number 
 between 15250000 and 15255000
 and "from" = '\xC098B2a3Aa256D2140208C3de6543aAEf5cd3A94'
 and value between 300000000000000000 and 360000000000000000
```
This gives me 96 txns, of which many had values like 0.3395ETH, 0.3495ETH, 0.3595ETH, and 0.3695ETH. Then I checked the addresses with the above txn values from FTX Exchange 2 and found that they all exhibit the same pattern, which was for interacting with zkSync, Aztec, and Safe, all of whom haven't issue tokens. So clearly this is airdrop farming.
